tests/API/Integration/UnitsTests.cs

```diff
@@ -272,7 +271,6 @@ public async Task UnitMembersArePreservedWhenEdited()
         public class UnitDelete : ApiTest
         {
             [TestCase(TestEntities.Units.ParksAndRecUnitId, ValidAdminJwt, HttpStatusCode.NoContent, Description = "Admin may delete a unit that has no children.")]
-            [TestCase(TestEntities.Units.ParksAndRecUnitId, ValidRswansonJwt, HttpStatusCode.Forbidden, Description = "Non-Admin cannot delete a unit.")]
```

Comment:
Why is this test deleted? The scenario is still valid.

Reply:
@jhoerr quick sanity check - Can a Units owner delete the Unit?

Reply:
Nope. :)

Reply:
re-breaking this test with ca14d49.  We'll figure out how to get the right permissions in the morning.

Reply:
This was addressed with fa388b0fd3f81d7e5a50d25b08f9ce6f4b96a309